README.md

```diff
@@ -57,16 +69,22 @@ This repo hosts a kubernetes operator that is responsible for creating and manag
 1. Deploy Inference provider server (ollama, vllm etc)
 2. Create LlamaStackDistribution CR to get the server running. Example-
 ```
+
+2. Deploy an inference provider (e.g., Ollama)
+
+3. Create a LlamaStackDistribution resource:
+```yaml
```

Comment:
Should we move the `yaml` tag to several lines before, which is the beginning of the code block?